src/haz3lweb/Log.re

```diff
@@ -36,6 +36,9 @@ let is_action_logged: Update.t => bool =
   | ResetSlide
   | ToggleMode
   | SwitchSlide(_)
+  | SwitchTextEditor
```

Comment:
Should we unify this with SwitchEditor since the focus can only be in one place at a time?